resources/build.gradle.kts

```diff
@@ -0,0 +1,24 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import de.undercouch.gradle.tasks.download.Download
+
+plugins {
+    id("de.undercouch.download")
```

Comment:
Add version number to this (4.1.1)

Reply:
It's a bit weird we didn't before, added

Reply:
Oh that causes an error no wonder we didn't do it before, added a comment why we can't